internal/cephfs/cephfs_util.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/ceph/ceph-csi/internal/util"
 )
 
+const MetadataPoolType = "metadata"
```

Comment:
@nixpanic just wondering are there different metadata types in general, otherwise its good to rename this to `PoolType`

Reply:
There are different types of pool, `data` and `metadata`. Calling this constant `PoolType` would not be clear (is it `data`, `metadata` or something else?).